app/_src/gateway/kong-enterprise/aws-iam-auth-to-rds-database.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Authenticate your Kong Gateway Amazon RDS database with AWS IAM
+badge: enterprise
+content_type: how-to
+---
+
+Starting in {{site.base_gateway}} 3.3.x, you can use AWS Identity and Access Management (IAM) authentication to connect to the AWS RDS database that you use for {{site.base_gateway}}. This page describes how to use this feature to secure your database configurations and database connections.
+
+When you enable this feature, you don't need to use a password when you connect to a database instance. Instead, you use a temporary authentication token. Because AWS IAM manages the authentication externally, the database doesn't store user credentials. If you use AWS RDS for {{site.base_gateway}}'s database, you can enable this feature on your running cluster. This ensures that you don't have to store database user credentials on both the {{site.base_gateway}} (`pg_password`) and RDS database side. 
+
+## AWS IAM authentication limitations
+
+AWS IAM authentication also has some limitations. Go through each one before you use this feature in your production environment:
+
+* For a traditional {{site.base_gateway}} cluster, only use IAM database authentication if {{site.base_gateway}} requires less than 200 new IAM database authentications per second. Establishing more connections per second can result in throttling. For more information, see [Recommendations for IAM database authentication](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html#UsingWithRDS.IAMDBAuth.ConnectionsPerSecond) in the Amazon RDS user guide. 
+* Enabling AWS IAM authentication requires SSL connection to the database. To do this, you must configure your RDS cluster correctly and provide the correct SSL-related configurations on {{site.base_gateway}}'s side. Enabling SSL also results in some performance overhead if you didn't previously use it. Currently, TLSv1.3 isn't supported by AWS RDS.
+- Since the Postgres RDS does not support mTLS, you can't enable mTLS between the {{site.base_gateway}} and the Postgres RDS database when AWS IAM authentication is enabled.
+- After enabling AWS IAM authentication in read-write or read-only mode, the `pg_password` or `pg_ro_password` configuration is ignored when establishing connections.
```

Comment:
I now feel this is not a limitation here, Can we move this to the enabling guide section and make it a note?

Reply:
@windmgc Looks like it's already a note in the "Enabling AWS IAM authentication" section, so I'll remove it from limitations and leave that existing note as-is.